gatsby-theme-shopify-manager/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "gatsby-theme-shopify-manager",
   "description": "The easiest way to build a Shopify shop on Gatsby.",
-  "version": "0.1.7",
+  "version": "0.1.8",
```

```suggestion
  "version": "0.1.7",
```
Let's put this back to the original version, since I typically add a version in separate commit.